com.unity.render-pipelines.universal/Shaders/PostProcessing/UpscaleSetup.shader

```diff
@@ -0,0 +1,50 @@
+Shader "Hidden/Universal Render Pipeline/Upscale Setup"
+{
+    HLSLINCLUDE
+        #pragma multi_compile_local_fragment _ _FXAA
+        #pragma multi_compile _ _USE_DRAW_PROCEDURAL
+
+        #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+        #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Filtering.hlsl"
+        #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+        #include "Packages/com.unity.render-pipelines.universal/Shaders/PostProcessing/Common.hlsl"
+
+        TEXTURE2D_X(_SourceTex);
+        float4 _SourceSize;
+
+        half4 Frag(Varyings input) : SV_Target
+        {
+            UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
+
+            float2 uv = UnityStereoTransformScreenSpaceTex(input.uv);
+            float2 positionNDC = uv;
+            int2   positionSS = uv * _SourceSize.xy;
+
+            half3 color = SAMPLE_TEXTURE2D_X(_SourceTex, sampler_PointClamp, uv).xyz;
+
+#if _FXAA
+            color = ApplyFXAA(color, positionNDC, positionSS, _SourceSize, _SourceTex);
+#endif
+
+            return half4(color, 1.0);
+        }
+
+    ENDHLSL
+
+    SubShader
```

Comment:
What is this Subshader user for? From what I understood if we don't have FXAA we don't need the upscale setup as the upscaling could be done in final post?

It would be nice to have some comments on the subshaders and usage for future maintainance / understanding.

Reply:
ScalingSetup makes sense to me.

Reply:
Ah I see. Ignore this comment for some reason I saw 2 subshaders. I'm crazy.